src/Dashboard/Dashboard.tsx

```diff
@@ -30,7 +25,16 @@ const Dashboard: React.SFC<DashboardProps> = (props) => {
 
 	useEffect(() => {getCocktail()}, [])
 
-	console.log(randomCocktail.strDrink, 'rando')
+	// const removeNulls = (info: RandomCocktail): RandomCocktail => {
```

Comment:
The function works in vanilla js but typescript is mad at the object.keys part. My mentor and I weren't able to get it working, but I did reach out to Hannah.